php/example_code/dynamodb/Readme.md

```diff
@@ -61,11 +67,18 @@ Or, to have the choices automatically selected, you can run it as part of a PHPU
 ..\..\vendor\bin\phpunit DynamoDBBasicsTests.php
 ```
 
-## Additional information
+### Prerequisites
+
+- You must have an AWS account, and have your default credentials and AWS Region configured as described in
+the [AWS Tools and SDKs Shared Configuration and Credentials Reference Guide]
+(https://docs.aws.amazon.com/credref/latest/refdocs/creds-config-files.html).
+- PHP 7.1 or later
+- Composer installed
+
+## Additional resources
 
 - [Amazon DynamoDB documentation](https://docs.aws.amazon.com/dynamodb)
```

Comment:
Not a deal-breaker, but the template asks for three links:
* [Service developer guide](link to developer guide)
* [Service API reference guide](link to developer guide)
* [SDK API reference guide](link to API reference guide)

Reply:
Done.